tests/deployment/requirements.txt

```diff
@@ -17,4 +17,6 @@
 # under the License.
 #
 
-https://repo.grakn.ai/repository/pypi-snapshot-group/packages/grakn-client/CLIENT_PYTHON_VERSION_MARKER/grakn-client-CLIENT_PYTHON_VERSION_MARKER.tar.gz
\ No newline at end of file
+--extra-index-url https://repo.grakn.ai/repository/pypi-snapshot/simple
+
+grakn-client==CLIENT_PYTHON_VERSION_MARKER
```

Comment:
`test-deployment-pip` makes use of this `requirements.txt` file to determine what to install in preparation for running the deployed artifact test.